src/primitives/useFocusManager.ts

```diff
@@ -99,10 +96,51 @@ const keyMapEntries: Record<string | number, string> = {
   Escape: 'Escape',
 };
 
+const keyHoldMapEntries: Record<string | number, string> = {
```

Comment:
This is cool - so users would specify which keys to check for hold - good optimization...